### ISSUE TITLE
There was an error during download.Failed to download container image

### ISSUE THREAD

**Issue by Marusyk** [Azure/service-fabric-issues#752]
When I try to use Docker containers in OnPremice Service Fabric cluster (6.0.219.9494) I'm getting

> Error event: SourceId='System.Hosting', Property='Download:1.0:1.0'.
> There was an error during download.Failed to download container image **xxx**

where **xxx** is any container name from Docker Hub (tried Linux and Windows containers and the result is the same). As I have already knew I can use only Windows container on Windows SF cluster but it doesn't work .

**Comment by Marusyk** [comment]
After run SF app with docker container I got

> Error response from daemon: open \\.\pipe\docker_engine_windows: The system cannot find the file specified.

when run `docker info` in console

**Comment by mani-ramaswamy** [comment]
This is likely because the download or extraction is timing out. Please try increasing the timeout as described in https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started-containers#configure-container-image-download-time and loop back.

**Comment by RajeetN** [comment]
What version of docker is running here on the nodes?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Marusyk** [comment]
On my machine installed 
![screenshot_1](https://user-images.githubusercontent.com/9221694/34845270-ffbfb01a-f71c-11e7-959d-774761b7cfe8.png)

**Comment by mgreen** [comment]
Hi @mani-ramaswamy. I'm experiencing the same issue on a hosted cluster. Do you know if it's possible to change the __ContainerImageDownloadTimeout__ via Custom fabric settings in the Portal? Or will I need to redeploy?

**Comment by Marusyk** [comment]
I increased the timeout. Did not help

**Comment by RajeetN** [comment]
Looks like you are running on windows 10? From documentation here https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-containers-overview

Note 

Deploying containers to a Service Fabric cluster in Windows 10 isn't supported yet. 

Also note that today, service fabric requires ee version of docker. This should have been clear in the documentation. We will fix it. Thanks!

-Rajeet

**Comment by Marusyk** [comment]
Ok thanks, I'm looking forward to windows 10 support

**Comment by dikoga** [comment]
I am having the same exactly issue running on a Windows 2016 Datacenter.

![image](https://user-images.githubusercontent.com/7737190/37530183-5fe36d5a-290f-11e8-9d09-706024993ddb.png)

**Comment by dikoga** [comment]
Hi @mani-ramaswamy , thank for you reply.

I have this configured now and I am getting the error bellow.

  <ServiceManifestImport>
    <ServiceManifestRef ServiceManifestName="Guest_Container1Pkg" ServiceManifestVersion="1.0.0" />
    <ConfigOverrides />
    <Policies>
      <ContainerHostPolicies CodePackageRef="Code">
        <RepositoryCredentials AccountName="digitalxpdev" Password="7Ps9TCY4STRq6E=xiesxPuB1ejhuhJh1" PasswordEncrypted="false"/>
        <PortBinding ContainerPort="80" EndpointRef="Guest_Container1TypeEndpoint"/>
        <ImageOverrides>
          <Image Name="digitalxpdev.azurecr.io/samples/webapidockerDefault"/>
          <Image Name="digitalxpdev.azurecr.io/samples/webapidocker1701" Os="14393" />
          <Image Name="digitalxpdev.azurecr.io/samples/webapidocker1709" Os="16299" />
        </ImageOverrides>
      </ContainerHostPolicies>
    </Policies>
  </ServiceManifestImport>

![image](https://user-images.githubusercontent.com/7737190/37538406-eb24fe58-2926-11e8-8826-d289f661f198.png)

I probably need to tag my container in the registry. However, not sure if I just tag the same image, create another, which tag, etc.. 

And, I could see many different files, traces, logs, etc. Which one can I get more information about what it is going on?

**Comment by MichalKoscielniak** [comment]
I'm having exactly the same issue as OP. I have just spinned new cluster using "Windows Server 2016 datacenter with Containers", i have created simple ASP .NET Core application ( using latest microsoft/aspnetcore image as base ). 
I RDP'ed to one of the SF VM and i can see folllowing 2 Warnings appearing in the EventLog all the time:
1. "Failed to import docker image error FABRIC_E_INVALID_OPERATION."
2. "ContainerImagesOperation returned FABRIC_E_INVALID_OPERATION, isDelete"

I tried to manually download my image on that environment. At the very end of download process it errors with this message:
"failed to register layer: re-exec error: exit status 1: output: ProcessUtilityVMImage C:\ProgramData\docker\windowsfilter\90832808765e5a389c8a58dba4130fa5af179ec169719c97ae66dc66809e251a\UtilityVM: The system cannot find the path specified."
